armi/physics/executers.py

```diff
@@ -70,7 +70,7 @@ def resolveDerivedOptions(self):
     def describe(self):
```

Comment:
Still wondering why this isn't just `__str__()`

Reply:
It's just so big! I'm always slightly surprised when objects str() is a gargantuan printout. But I guess that may be a me problem.